build/openwisp_dashboard/urls.py

```diff
@@ -14,9 +18,27 @@
     url(r'^accounts/', include('openwisp_users.accounts.urls')),
     url(r'^$', redirect_view, name='index'),
     url(r'^', include('openwisp_radius.urls', namespace='freeradius')),
+    url(r'^topology/', include(visualizer_urls)),
 ]
 
-urlpatterns += openwisp_controller_urls()
-urlpatterns += openwisp_topology_urls()
+def topology_url(urlpatterns, pattern, receive_name):
+    receive_url = url(pattern,
+                      RedirectView.as_view(url=os.environ['TOPOLOGY_DOMAIN']),
+                      name=receive_name)
+    urlpatterns.append(receive_url)
+
+for api_url in topology_urls:
+    if hasattr(api_url, 'name') and api_url.name == receive_name:
+        regex = str(api_url.pattern)
+        topology_url(urlpatterns, regex, receive_name)
+        break
+    elif api_url.url_patterns:
+        for path in api_url.url_patterns:
+            if hasattr(path, 'name') and path.name == receive_name:
+                path_regex = str(path.pattern)[1:] if str(path.pattern)[0] == '^' else str(path.pattern)
+                regex = str(api_url.pattern) + path_regex
+                topology_url(urlpatterns, regex, receive_name)
+                break
+
```

Comment:
What do you think about this solution? 
I am copying the path from `topology_urls.py` file and registering it in like: 
```
    receive_url = url(pattern,
                      RedirectView.as_view(url=os.environ['TOPOLOGY_DOMAIN']),
                      name=receive_name)
```
make sure that dashboard gets the path while the url points to topology container. :smile: